SanaraV3/StaticObjects.cs

```diff
@@ -225,6 +226,11 @@ public static async Task InitializeAsync(Credentials credentials)
             {
                 Website = new UploadManager(credentials.StatsWebsiteUrl, credentials.StatsWebsiteToken);
             }
+
+            if(credentials.MyDramaListApiKey != null)
```

Comment:
Please add a space between the if and the parenthesis
```suggestion
            if (credentials.MyDramaListApiKey != null)
```